src/Runner.Common/JobServerQueue.cs

```diff
@@ -230,21 +232,31 @@ public void QueueFileUpload(Guid timelineId, Guid timelineRecordId, string type,
             _fileUploadQueue.Enqueue(newFile);
         }
 
-        public void QueueSummaryUpload(Guid stepRecordId, string name, string path, bool deleteSource)
+        public void QueueResultsUpload(Guid recordId, string name, string path, string type, bool deleteSource, bool finalize, bool firstBlock, long totalLines)
         {
+            if (recordId == _jobTimelineRecordId) 
+            {
+                Trace.Verbose("Skipping job log {0} for record {1}", path, recordId);
+                return;
+            }
+
```

Comment:
should we also return early if `_resultsClientInitiated == false`?

Reply:
Sure, doesn't hurt to guard here as well.